### DISCUSSION TITLE
Application denied without reason

### DISCUSSION THREAD

**Discussion by fakuuy** [community/community#134655] [GitHub Education]
Hi! I'm trying to get the student pack and every time that I try to send the application gets denied. I'm attaching the document that I use to get verified and works with others developers like Autodesk so I don't know what's the problem

![image](https://github.com/user-attachments/assets/fd2d2a28-7197-4d03-a41a-2c5fdcb0c195)

**Comment by PstasDev** [comment]
Hi! You should really try these options and sending your application again:

- Fix the orientation of the picture, rotate it by 90 degrees
- If you have one, use a picture of a Student ID card instead
- Make sure that the picture is in the best allowed quality (doesn’t max out the file size limit, but reaches the pixel minimums)

These are essential steps, as **_there is an automation that’s handling most of the requests instead of human officials_**.
